intrahospital_api/management/commands/deployment_check.py

```diff
@@ -0,0 +1,42 @@
+"""
+A management command that looks at the last x hours
+(by default 4) and makes sure the batch load
+and the inital load all load are all the same.
+
+This should only run on test and will blow up if you
+try and run it outside of test
+"""
+
+import datetime
+import six
+import json
+from django.utils import timezone
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from intrahospital_api import deployment_check
+
+
+class Command(BaseCommand):
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'hours',
+            help="hours ago",
+            type=int,
+            default=4
+        )
+
+    def handle(self, *args, **options):
+        result = {}
```

Comment:
I'm not really sure why we're passing in this dictionary ? Can't we just return from a function ?